common/workunit/workunit.cpp

```diff
@@ -14243,6 +14243,15 @@ static void runKubectlCommand(const char *title, const char *cmd, const char *in
     }
 }
 
+bool checkK8sEndpoints(const char *serviceName)
```

Comment:
would isActiveK8sService() (or similar) be a better name - since it returns a boolean?